code/game/machinery/drone_dispenser.dm

```diff
@@ -105,7 +105,7 @@
 	dispense_type = list(/obj/effect/mob_spawn/ghost_role/drone/syndrone/badass)
 	end_create_message = "dispenses an ominous suspicious drone shell."
 
-// I don't need your forgiveness, this is awesome.
```

Comment:
Find and replace moment

Reply:
tools don't need your forgiveness either